### PR TITLE
Update package.json

### PR DESCRIPTION
**Pull request by Hyra**
Added missing deps

### CODE REVIEW

**Comment by arunoda** [comment]
Are you using NPM2 ? In NPM3 this is not an issue.

File: package.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
   "dependencies": {
     "babel-polyfill": "^6.3.14",
     "react": "^0.14.7",
-    "react-dom": "^0.14.7"
+    "react-dom": "^0.14.7",
+    "express": "^4.13.4",
```
Comment (arunoda) [inline package.json:14]: Express should be dev dependency

**Comment by Hyra** [comment]
Yeah, `npm 2.14.4`

Maybe add an engine to the package?

```
{ "engines" : { "npm" : ">=3.0.0" } }
```

**Comment by arunoda** [comment]
That's a great idea. Could you send us a PR. (to the storybook package)

**Pull request closed without merging**